[Ready for Review] Update BinderHub workshop (#1552)
alan-turing-institute/the-turing-way: workshops/build-a-binderhub/binderhub_resources/collaborative_notes_template.md

> [comment]: <This is a templatefor taking collaborative notes during a Build a BinderHub workshop. It has been formatted to be Markdown compatible and should "just work" when copy-pasted into HackMD.io>

Reviewer: Typo: 'templatefor'

> [comment]: <This is a templatefor taking collaborative notes during a Build a BinderHub workshop. It has been formatted to be Markdown compatible and should "just work" when copy-pasted into HackMD.io>

Author: ```suggestion
[comment]: <This is a template for taking collaborative notes during a Build a BinderHub workshop. It has been formatted to be Markdown compatible and should "just work" when copy-pasted into HackMD.io>
```

# Notes from Build a BinderHub workshop

[comment]: <Placeholder for the date, time, name and location of the workshop or conference.>

- Date:
- Time:
- Location:

## IMPORTANT

If you have not done so already, please register for:

[comment]: <Select either the Azure Pass or Free Trial depending on Microsoft support at the event>
[comment]: <A platform such as Slack may be a good way to collect emails for the Azure Pass. Link to the appropriate channel here.>

* a Docker Hub account: <https://hub.docker.com/signup>
* And **EITHER/OR**:
  * a Microsoft Azure Pass
    * If possible, please **do not** use a ".ac.uk" domain address.
  * a Microsoft Azure Free Trial: <https://azure.microsoft.com/en-gb/free/>

If you'd like to use this HackMD pad anonymously, that's totally fine! Just sign in as a guest :smile:

## Useful links

[comment]: <It's useful to create a short bit.ly link for the hackmd pad>
* This HackMD pad:

> * workshop instructions: [bit.ly/zero-to-binderhub-workshop](https://bit.ly/zero-to-binderhub-workshop)

Reviewer: Typo: 'w' should be capitalised?

> * workshop instructions: [bit.ly/zero-to-binderhub-workshop](https://bit.ly/zero-to-binderhub-workshop)

Author: ```suggestion
* Workshop instructions: [bit.ly/zero-to-binderhub-workshop](https://bit.ly/zero-to-binderhub-workshop)
```

[comment]: <If there are slides accompanying the workshop, link to them here>
* The intro slides:
* The Turing Way repo: [alan-turing-institute/the-turing-way](https://github.com/alan-turing-institute/the-turing-way)

> * Chat with us on Gitter!

Reviewer: Include a link to Turing Way slack channel?

> * Chat with us on Gitter!

Author: ```suggestion
* Chat with us on [Slack](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw)!
```
